number_guesser.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+
+def main():
+    number_guess()
+
+
+def number_guess():
+    """a function to generate a random number and has the user guess"""
+    test = 0
+    user_num = int(input("Guess a whole number(Non negative) from 1 to 100: \
+                            "))
+    randnum = random.randint(1, 100)
+    while test == 0:
+        if randnum == user_num:
+            print("You're correct!")
```

Comment:
I like the `print "you're correct!"` syntax a bit more than "print()" syntax, I'm not sure what the best practice on this is.